x/poe/bootstrap.go

```diff
@@ -89,14 +89,52 @@ func bootstrapPoEContracts(ctx sdk.Context, k wasmtypes.ContractOpsKeeper, tk tw
 	}
 	ocInitMsg := newOCInitMsg(gs)
 	deposit := sdk.NewCoins(gs.OversightCommitteeContractConfig.EscrowAmount)
-	ocContractAddr, _, err := k.Instantiate(ctx, ocCodeID, systemAdminAddr, systemAdminAddr, mustMarshalJson(ocInitMsg), "oversight_committee", deposit)
+
+	ocMembers := make([]string, len(gs.Engagement))
```

Comment:
The OversightCommunity (OC) is not the same as the engagement addresses. Although there can be some overlap. 
We would need a new element in genesis to define the addresses for all OC members.